### ISSUE TITLE
Make notification time configurable

### ISSUE THREAD

**Issue by flocke** [saschpe/BirthdayCalendar#5]
I really love this app but I would like to have the option to define the time I get notified of birthdays. Currently you only can set it to notify you on the same day and one before, but not the time this notification occurs.

**Comment by saschpe** [comment]
Hi,
Good point. I have to think about it a little ;-)
--
Mit freundlichen Grüßen,
Sascha Peilicke

**Comment by salim-b** [comment]
May I chime in here and place an associated feature reguest?

Currently it's "only" possible to configure two reminders. I'd love to be able to configure _three_ reminders (in my case: same day, 7 days ahead and 14 days ahead). Could a third reminder option be added? Or maybe the reminders could even be made "dynamic", so people can add as many reminders as they want to.

**Comment by bexale** [comment]
Yes, it's a problem that the reminder with the notification  arrives at 00:00 AM (midnight).

I switch off completely my phone to sleep well every night... So, when I switch on my phone the morning, I don't reveive the notification...

So, the app is not useful in this case...

Thank you Sascha to fix this problem :-)

**Comment by PanderMusubi** [comment]
Yes, please let the use choose a notification (e.g. 09:00) and a snooze time (e.g. one hour).
